spec/helpers/application_helper_spec.rb

```diff
@@ -3,7 +3,8 @@
 describe ApplicationHelper do
   describe '#display_url' do
     it 'returns the domain portion of a given url' do
-      expect(helper.display_url('http://www.foo.com/bar/12')).to eq('www.foo.com/bar/12')
+      expect(helper.display_url('http://www.foo.com/bar/12'))
+        .to eq('www.foo.com/bar/12')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.